runSyst.sh

```diff
@@ -0,0 +1,9 @@
+YEAR=2018
```

Comment:
remove `runSyst.sh` and `runAllYears` from the root directory since they are now in `scripts/`